examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/container/.content.xml

```diff
@@ -4,4 +4,5 @@
     jcr:title="Adaptive Form Container"
     jcr:description="Add an Adaptive Form to a page."
     sling:resourceSuperType="core/fd/components/form/container/v2/container"
+    cq:styleElements="[div,section,article,main,aside,header,footer]"
```

Comment:
why do we need `cq:styleElements` for this particular change?

Reply:
@deepprakash345 - Since we are changing main tag in page, we can ignore this changes.

@nit23uec - style element is good have capabilities on container as it provides flexibility  to change container tag type without any new component. Site included in there example

Reply:
but this is the form container. why would anyone want to have a different tag on the form container other than "form"?

Reply:
done